spec/lib/schema_file_loader_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "rails_helper"
+
+# TODO: .new is only needed if we are testing instance access instead of table access.
+#   Consider removing .new in specs below unless necessary
+
+describe SchemaFileLoader do
+
+  it "all required schema files are present" do
+    expect(SchemaFileLoader::EFILE_SCHEMAS_FILENAMES).to eq [
+      ["efile1040x_2020v5.1.zip", "irs"],
+      ["efile1040x_2021v5.2.zip", "irs"],
+      ["efile1040x_2022v5.3.zip", "irs"],
+      ["efile1040x_2023v5.0.zip", "irs"],
+      ["NYSIndividual2023V4.0.zip", "us_states"],
+      ["AZIndividual2023v1.0.zip", "us_states"],
+    ]
+  end
+
+  context "s3_credentials" do
+
+    context "AWS_ACCESS_KEY_ID in ENV" do
+      it "uses the environment variables" do
+        stub_const("ENV", {
+          "AWS_ACCESS_KEY_ID" => "mock-aws-access-key-id",
+          "AWS_SECRET_ACCESS_KEY" => "mock-aws-secret-access-key"
```

Comment:
very small syntax thing: i think we tend to favor `{AWS_ACCESS_KEY_ID: "mock-aws-access-key-id"}` over `{"AWS_ACCESS_KEY_ID" => "mock-aws-access-key-id"}`

Reply:
I found the code in the AWS client actually looks for strings here rather than symbols, so the : syntax did not work.